openshift/dotnet-webapi.dc.yaml

```diff
@@ -116,8 +117,8 @@ objects:
       name: '${NAME}${SUFFIX}'
     spec:
       ports:
-        - name: '5000'
-          port: 5000
+        - name: '8080'
+          port: 8080
```

Comment:
Why is this 8080 now? what changed?

Reply:
I'm going to revert this.  I was updating endpoints for URIs to the API but forgot to revert.